test/e2e/config/cloudstack.yaml

```diff
@@ -129,7 +132,8 @@ variables:
   CLOUDSTACK_IMPOSSIBLE_CONTROL_PLANE_MACHINE_OFFERING: "Impossible Instance"
   CLOUDSTACK_WORKER_MACHINE_OFFERING: "Medium Instance"
   CLOUDSTACK_IMPOSSIBLE_WORKER_MACHINE_OFFERING: "Impossible Instance"
-  CLOUDSTACK_TEMPLATE_NAME: kube-v1.20.10/ubuntu-2004
+  CLOUDSTACK_TEMPLATE_NAME: kube-v1.22.6/ubuntu-2004
```

Comment:
I believe these will break our current e2e, as we don't have this template on our ACS configuration.  We'll need to adapt it accordingly.

Reply:
New templates can be found here
http://packages.shapeblue.com/cluster-api-provider-cloudstack/images/